tests/test.py

```diff
@@ -56,7 +57,10 @@ def _require_package(pkg_name):
     pkg_manager = dcos.package.get_package_manager()
     installed_pkgs = dcos.package.installed_packages(pkg_manager, None, None, False)
     if not any(pkg['name'] == pkg_name for pkg in installed_pkgs):
-        shakedown.install_package(pkg_name, wait_for_completion=True)
+        options = {}
```

Comment:
This isn't guaranteed to be correct for an arbitrary package, so let's not include these options in this function.  Pass them as an argument to the function through `_require_package('hdfs')`.

Also, let's create the options functionally:
```
if (...):
  options = ...
else:
  options = ...
```